css/styles.css

```diff
@@ -80,13 +106,51 @@ input.notes-input {
   object-fit: cover;
 }
 
-.img-container {
+.img-container-create {
   width: 100%;
   aspect-ratio: 1 / 1;
 }
 
+.img-container-view {
+  width: 100%;
+  aspect-ratio: 16 / 11;
```

Comment:
What is this doing? You would likely be better off using something like the `object-fit` property:

https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit